app/forms/work_form.rb

```diff
@@ -42,4 +42,24 @@ def license_label
     selected_license = WorkForm.licenses.select { |_, v| v['uri'] == license }
     selected_license.keys.first
   end
+
+  attribute :work_type, :string
+  validates :work_type, presence: true, if: :deposit?
+
+  attribute :work_subtypes, array: true, default: -> { [] }
+  # Music and mixed materials have a minimum number of subtypes required
+  validates :work_subtypes, length: { minimum: WorkType::MINIMUM_REQUIRED_MUSIC_SUBTYPES,
+                                      too_short: '%<count>s term is the minimum allowed' },
```

Comment:
Would this end up being something like `2s term is the minimum allowed` instead of something like `2 terms is the minimum allowed`?